pkg/resources/configmaps/prometheus.go

```diff
@@ -80,22 +80,29 @@ func GetPrometheusConfig(t *miniov2.Tenant, accessKey, secretKey string) *Promet
 			ScrapeInterval:     miniov2.MinIOPrometheusScrapeInterval,
 			EvaluationInterval: 30 * time.Second,
 		},
-		ScrapeConfigs: []ScrapeConfig{
-			{
-				JobName:     t.PrometheusConfigJobName(),
-				BearerToken: bearerToken,
-				MetricsPath: miniov2.MinIOPrometheusPathCluster,
-				Scheme:      minioScheme,
-				TLSConfig: tlsConfig{
-					CAFile: "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
-				},
-				StaticConfigs: []staticConfig{
-					{
-						Targets: []string{minioTargets},
-					},
+		ScrapeConfigs: []ScrapeConfig{},
+	}
+
+	if len(t.Spec.PrometheusOperatorScrapeMetricsPath) == 0 {
+		t.Spec.PrometheusOperatorScrapeMetricsPath = []string{"/minio/v2/metrics/cluster"}
+	}
+
+	for index, scrape := range t.Spec.PrometheusOperatorScrapeMetricsPath {
+		promConfig.ScrapeConfigs = append(promConfig.ScrapeConfigs, ScrapeConfig{
```

Comment:
Consider the following tenant yaml section; note that v3 cluster metrics is scraped using `/minio/v3/metrics/cluster/` (see https://github.com/minio/minio/blob/master/docs/metrics/v3.md):
```
  prometheusOperator: true
  prometheusOperatorScrapeMetricsPath:
  - /minio/v2/metrics/bucket
  - /minio/v2/metrics/cluster
  - /minio/v2/metrics/node
  - /minio/v2/metrics/resource
  - /minio/v3/metrics/cluster/
```

v3 metrics scraping is failing with `server returned HTTP status 403 Forbidden`. Please try and check this.